specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/table.json

```diff
@@ -164,6 +164,9 @@
         "x-ms-examples": {
           "TableOperationPut": {
             "$ref": "./examples/TableOperationPut.json"
+          },
+          "TableOperationPutOrPatchAcls": {
+            "$ref": "./examples/TableOperationPutOrPatchAcls.json"
```

Comment:
This API input parameter only has "#/parameters/TableName", so the swagger not support put  table with signedIdentifiers.

Reply:
Added